readme.md

```diff
@@ -350,6 +350,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[AwesomeTechStack](https://awesometechstack.com)** — AwesomeTechStack is a free to use website tech stack analyzer. AwesomeTechStack provides insights into the security, modernity, and performance of any website's technology stack and guidance to improve performance. Lighthouse insights are a crucial part of a website's tech stack rating.
 
+* **[Wattspeed](https://wattspeed.com/)** — Wattspeed is a FREE tool that runs snapshots, historical captures of your web pages. Wattspeed includes Lighthouse scores, technologies list, W3C HTML validator results, DOM size & mixed content info, and more.
+
```

Comment:
```suggestion
* **[Wattspeed](https://wattspeed.com/)** — Wattspeed is a free tool that runs snapshots, historical captures of your web pages. Wattspeed includes Lighthouse scores, a technologies list, W3C HTML validator results, DOM size, mixed content info, and more.
```

Reply:
^ Suggested edit for the text. Can you clarify what `runs snapshots` means, though? Is the Wattspeed result (Lighthouse scores, technologies list, validator results, etc) a snapshot, or is it taking snapshots of web page content and producing those things later?

Reply:
> [Wattspeed](https://wattspeed.com/)** — Wattspeed is a free tool that runs snapshots -  historical captures of your web pages that include Lighthouse scores, a list of technologies, W3C HTML validator results, DOM size, mixed content info, and more.

Thanks for your feedback!
Does the above explain better what a snapshot is in this case?

Reply:
Ah, great, thanks. I'd maybe tweak it to say "produces snapshots" (or something like that) instead of "runs snapshots", but otherwise looks great!

Reply:
Thanks a lot. 
I did use the "generates snapshots" wording and made a new fresh commit.